mat64/index_bound_checks.go

```diff
@@ -40,7 +40,8 @@ func (m *Dense) set(r, c int, v float64) {
 	m.mat.Data[r*m.mat.Stride+c] = v
 }
 
-// At returns the element at row r, column c. It panics if c is not zero.
+// At returns the element at row r, column c.
```

Comment:
While we're here, do we want to make this At(i,j int)? Frequently r and c and the number of rows and columns, and i and j are a specific index.